### DISCUSSION TITLE
SMTP Relay Option?

### DISCUSSION THREAD

**Discussion by turbogizzmo** [berrystephenw/OctoText#132] [Q&A] [answered]
Does this support pointing to a local smtp relay to email out? 

I have tried with just the email account and no password but getting auth errors but we do have other devices on the network able to relay out.

**Comment by berrystephenw** [comment, marked as answer]
Have you tried using the "user name only authentication" button? 
![image](https://user-images.githubusercontent.com/44931130/204149157-890ecbaf-aa8f-47c3-8926-e35ba6807dac.png)

**Comment by turbogizzmo** [comment]
Yes the solution was just the email, no check box, no password. Also there is a delay in emails getting to @vtext.com and @vzwpix.com so that could have confusing/complicating troubleshooting.
